plugins/queue/scheduler.js

```diff
@@ -203,13 +207,15 @@ async function startPeriodic(executor, config) {
             // eslint-disable-next-line max-len
             if (err && err.message !== 'Job already enqueued at this time with same arguments') {
                 shouldRetry = true;
+                logger.error(`duplicate build: failed to enqueue for job ${job.id}: ${err}`);
             }
```

Comment:
`else` should log the error since it's not a retry case and we failed to enqueue job